src/jit/importer.cpp

```diff
@@ -7715,7 +7715,7 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
                         unsigned   calliSlot  = lvaGrabTemp(true DEBUGARG("calli"));
                         LclVarDsc* varDsc     = &lvaTable[calliSlot];
                         varDsc->lvVerTypeInfo = tiRetVal;
-                        impAssignTempGen(calliSlot, call, clsHnd, (unsigned)CHECK_SPILL_NONE);
+                        impAssignTempGen(calliSlot, call, tiRetVal.GetClassHandle(), (unsigned)CHECK_SPILL_NONE);
```

Comment:
Ah, I see. So the `clsHnd = calliSig.retTypeClass;` was only added to support the line I'm changing anyway (in #9191). Line removed.

The `sig->retTypeClass` is only a question of whether `NormaliseForStack`, `SetIsReadonlyByRef`, `SetIsPermanentHomeByRef` et al mess with the class handle in any interesting way. It seems like they don't (so the value will be the same).

It seems reasonable to use `tiRetVal` to obtain the value since that's what we also use to initialize `varDsc->lvVerTypeInfo` line above.